test/api/handler_test.clj

```diff
@@ -14,39 +14,37 @@
 (defn new-game [] (handler/handler (mock/request :post "/lobby")))
 (defn join-game [game-id] (handler/handler (mock/request :post (string/join ["/games/" game-id]))))
 
+(defn check-response
```

Comment:
Minor gripe: this function might be more descriptive as `expect-response` since we use `expect` to assert things in tests and that's all this function does